RussianFilter/sections/specific.txt

```diff
@@ -51,6 +51,7 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 !
 !
 !
+rutracker.org##div[class^="bn-"]
```

Comment:
Not reproduced on my end - we already have rules for this site in AdGuard Russian. Can you send a screenshot with ad elements and opened devtools?
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/130577499-d1230683-7e30-4fba-b802-d2d4f09846d6.png)

</details><br/>

Reply:
@gork7777 according to previous pull requests (PR's), please add one pr = one issue.
For the rutracker we have common rules with mirrors, you also have to check it next time.